src/components/header/SelectionScrubberOverlay.js

```diff
@@ -2,14 +2,41 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
 import React, { PureComponent, PropTypes } from 'react';
 import classNames from 'classnames';
 import clamp from 'clamp';
 import Draggable from '../shared/Draggable';
 import { getFormattedTimeLength } from '../../profile-logic/range-filters';
+import { updateProfileSelection } from '../../actions/profile-view';
```

Comment:
I'd prefer we don't import it if we don't use it directly -- here because we use it only for its type, I think we're tying too much the interface and the implementation.